### ISSUE TITLE
Contribution in Project

### ISSUE THREAD

**Issue by vishalmaurya850** [Vijayvarma115/bluestock-ipo-webapp#1]
Uu

**Comment by Vijayvarma115** [comment]
Already we have done 50-60% can you continue and I will give you thr
required resources it's a company's interns project

**Comment by vishalmaurya850** [comment]
Yeah Okay I will contribute to this project and just for information will I get something for this project?

**Comment by vishalmaurya850** [comment]
Yeah Okay I will contribute to this project and just for information will I get something for this project?
